tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -47,69 +47,61 @@ def namespace(self, project_factory):
-            sc_name=self.sc_obj,
-            namespace=self.namespace,
-            number_of_pvc=number_of_pvc,
-            size=f"{random.randrange(5, 105, 5)}Gi",
-            access_mode=access_mode
+        # Get pvc_dict_list, append all the pvc.yaml dict to pvc_dict_list
+        pvc_dict_list1 = scale_lib.construct_pvc_creation_yaml_bulk_for_kube_job(
+            no_of_pvc=number_of_pvc, access_mode=access_mode, sc_name=sc_name
+        )
+        pvc_dict_list2 = scale_lib.construct_pvc_creation_yaml_bulk_for_kube_job(
+            no_of_pvc=number_of_pvc, access_mode=access_mode, sc_name=sc_name
         )
 
-        # Check for PVC status using threads
-        threads = list()
-        for obj in pvc_objs:
-            process = threading.Thread(
-                target=helpers.wait_for_resource_state,
-                args=(obj, constants.STATUS_BOUND, )
-            )
-            process.start()
-            threads.append(process)
-        for process in threads:
-            process.join()
+        # There is 2 kube_job to reduce the load, observed time_out problems
+        # during delete process of single kube_job and heavy load.
+        job_file1 = ObjectConfFile(
+            name='job_profile_1', obj_dict_list=pvc_dict_list1,
+            project=self.namespace, tmp_path=tmp_path
+        )
+        job_file2 = ObjectConfFile(
+            name='job_profile_2', obj_dict_list=pvc_dict_list2,
+            project=self.namespace, tmp_path=tmp_path
+        )
 
-        # Get pvc_name, require pvc_name to fetch creation time data from log
-        threads = list()
-        for pvc_obj in pvc_objs:
-            process = threading.Thread(target=pvc_obj.reload)
-            process.start()
-            threads.append(process)
-        for process in threads:
-            process.join()
+        # Create kube_job
+        job_file1.create(namespace=self.namespace)
+        job_file2.create(namespace=self.namespace)
 
-        pvc_name_list, pv_name_list = ([] for i in range(2))
-        threads = list()
-        for pvc_obj in pvc_objs:
-            process1 = threading.Thread(target=pvc_name_list.append(pvc_obj.name))
-            process2 = threading.Thread(target=pv_name_list.append(pvc_obj.backed_pv))
-            process1.start()
-            process2.start()
-            threads.append(process1)
-            threads.append(process2)
-        for process in threads:
-            process.join()
+        # Check all the PVC reached Bound state
+        pvc_bound_list = scale_lib.check_all_pvc_reached_bound_state_in_kube_job(
+            kube_job_obj=job_file1, namespace=self.namespace, no_of_pvc=number_of_pvc
+        )
+        pvc_bound_list.extend(
+            scale_lib.check_all_pvc_reached_bound_state_in_kube_job(
+                kube_job_obj=job_file2, namespace=self.namespace, no_of_pvc=number_of_pvc
+            )
+        )
+
+        logging.info(f"Length of pvc_bound_list {len(pvc_bound_list)}")
```

Comment:
"Length of pvc_bound_list" can be "Number of PVCs bound" for better understanding.

Reply:
ack.